airbyte-integrations/connectors/source-postgres/README.md

```diff
@@ -0,0 +1,13 @@
+# How to use Postgres Script?
```

Comment:
Similar as MsSQL, it's better to put this under a performance test H2 title.

```suggestion
# Postgres Source

## Performance Test

### Use Postgres script to populate the benchmark database
```

Reply:
updated it